code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -108,6 +112,18 @@
 	SIGNAL_HANDLER
 	current_order = order
 
+
+///Receive fire support laser notifications
+/mob/living/silicon/ai/proc/receive_laser_OB(datum/source, obj/effect/overlay/temp/laser_target/OB/incoming_laser)
+	SIGNAL_HANDLER
+	to_chat(src, "<span class='notice'>Orbital Bombardment laser detected. Target: [AREACOORD_NO_Z(incoming_laser)] </span>")
+	playsound(src, 'sound/effects/binoctarget.ogg', 15)
+
+/mob/living/silicon/ai/proc/receive_laser_CAS(datum/source, obj/effect/overlay/temp/laser_target/cas/incoming_laser)
```

Comment:
Do not use upper case in proc names like this

Reply:
Applied

Reply:
Applied